python_parser/check_summary_quality.py

```
@@ -0,0 +1,211 @@
+                    })
+    rows.sort(key=lambda r: (-r["score"], r["book"], r["chapter"], r["startVerse"]))
+    return rows
+
+
+def print_report(rows: list[dict], threshold: float) -> None:
+    if not rows:
+        print(f"No passage summaries exceed the {threshold:.0%} overlap threshold. ✨")
+        return
+
+    print(f"\nFound {len(rows)} passage summaries with ≥{threshold:.0%} word overlap")
+    print(f"against their verse text (sorted by severity).\n")
```

Ruff (0.15.9)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)